firebase.json

```diff
@@ -1,12 +1,10 @@
 {
   "hosting": {
-    "public": "build",
-    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
-    "rewrites": [
-      {
-        "source": "**",
-        "destination": "/index.html"
-      }
```

Comment:
Was removing this necessary?  Just wondering

Reply:
I did not remove this haha, I think when I set up firebase on my side it overrode this file.